js/canvasmanager.ts

```diff
@@ -12,27 +12,76 @@ interface CanvasManagerEvents {
   change: () => void;
 }
 
-interface Event {
-  undo: (cm: CanvasManager) => void;
+// Represents arguments of array splice method.
```

Comment:
Could we make this a JSDoc comment `/** ... */` so that it shows up in IDE autocomplete suggestions? (and all other comments introduced in this commit that are meant to document what something is/does)